src/win/fs.c

```diff
@@ -2009,28 +2010,146 @@ static void fs__ftruncate(uv_fs_t* req) {
   }
 }
 
+static BOOL fs__clonefile_dup_extents(HANDLE src, HANDLE dst, int64_t offset, int64_t clone_size) {
```

Comment:
Long line, should be <= 80 columns. Most idiomatically fixed by lining up the arguments on separate lines.

Reply:
```suggestion
static BOOL fs__clonefile_dup_extents(HANDLE src, 
                                      HANDLE dst, 
                                      int64_t offset,
                                      int64_t clone_size) {
```